ja/news/_posts/2019-04-01-ruby-2-4-6-released.md

```diff
@@ -25,21 +25,25 @@ Ruby 2.4.6 がリリースされました。
 ## ダウンロード
 
 * [https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2](https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2)
```

Comment:
We also use this shorthand syntax for links:

```
<https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2>
```

Reply:
ok